fix name of FUSE library to search for (libfuse3.so for FUSE 3.x)
author: Kenneth Hoste (HPC-UGent)
updated to 3.0.1
author: J. Sassmannshausen (Imperial College London/UK)
diff --git a/fusepy-3.0.1.orig/fuse.py b/fusepy-3.0.1/fuse.py
index 7c2ddc4..f95730f 100644
--- a/fusepy-3.0.1.orig/fuse.py
+++ b/fusepy-3.0.1/fuse.py
@@ -109,7 +109,7 @@ if not _libfuse_path:
         if _libfuse_path:
             _libfuse_path += r"bin\winfsp-%s.dll" % ("x64" if sys.maxsize > 0xffffffff else "x86")
     else:
-        _libfuse_path = find_library('fuse')
+        _libfuse_path = find_library('fuse3')
 
 if not _libfuse_path:
     raise EnvironmentError('Unable to find libfuse')
